spec/builtin/ssl/ip_range_enum_spec.rb

```diff
@@ -0,0 +1,28 @@
+require 'spec_helper'
+require 'ronin/recon/builtin/net/ip_range_enum'
+
+describe Ronin::Recon::Net::IPRangeEnum do
+  describe "#process" do
+    context "when there are ip in range" do
+      let(:ip_range) { Ronin::Recon::Values::IPRange.new('192.168.0.1/2') }
```

Comment:
`192.168.0.1/2` will yield *many* IPs. You probably want `192.168.0.1/30` to only enumerate the last two bits of the IP address.